scst-store/multicluster-setup.hbs.md

```diff
@@ -109,36 +98,21 @@ Run cluster:
 * CloudEvent Handler CA certificate
 * CloudEvent Handler edit token
 
-### <a id='apply-kubernetes'></a>Apply the Metadata Store CA certificate and authentication token to the Build cluster
-
-Before you deploy the Build profile, apply the Metadata Store CA certificate and
-Metadata Store authentication token from the earlier steps. This gives the Build profile deployment access to these values.
-
-To apply the Metadata Store CA certificate and an authentication token:
-
-1. With your kubectl targeted at the Build cluster, create a namespace for the Metadata Store CA
-certificate and authentication token.
+### <a id='apply-kubernetes'></a>Configure SCST - Scan with the Metadata Store CA certificate and authentication token on the Build cluster
 
-    ```console
-    kubectl create ns metadata-store-secrets
-    ```
-
-1. Apply the Metadata Store CA certificate `store_ca.yaml` secret YAML you generated earlier.
-
-    ```console
-    kubectl apply -f store_ca.yaml
-    ```
-
-2. Create a secret to store the access token. This uses the Metadata Store `MDS_AUTH_TOKEN` environment variable.
+1. Update the Build profile `values.yaml` file to add the following snippet using the contents of $MDS_CA_CERT and $MDS_AUTH_TOKEN copied in an earlier step. It configures SCST - Scan with the Metadata Store CA certificate and authentication token.
```

Comment:
You can remove the number since this list only has one item

Reply:
thanks!! removed